common/app/views/fragments/items/facia_cards/contentCard.scala.html

```diff
@@ -19,21 +18,25 @@
         data-discussion-closed="@item.discussionSettings.isClosedForComments"
         data-discussion-url="@item.header.url.get(request)#comments"
     }
+    data-link-name="@item.analyticsPrefix | card-@{ index + 1 }"
+    data-item-visibility="@visibilityDataAttribute"
+    data-test-id="facia-card"
+    @item.id.map { id => data-id="@id" }
+    @item.snapStuff.map(_.dataAttributes)
+    @item.shortUrl.map { shortUrl => data-loyalty-short-url="@shortUrl" }>
 
-data-link-name="@item.analyticsPrefix | card-@{
```

Comment:
I moved it to https://github.com/guardian/frontend/pull/11635/files#diff-a2b2ea169f137c124dce5857de42402fR21